modules/gis/src/index.d.ts

```diff
@@ -1,5 +1,5 @@
 // Types
-export {BinaryGeometryData, BinaryAttribute} from './types';
+export {BinaryGeometryData, BinaryAttribute} from './types.d.ts';
```

Comment:
Interesting. I would much prefer we changed these two files to `.ts` and used `export type {...`

Reply:
> Interesting. I would much prefer we changed these two files to `.ts` and used `export type {...`

Yes, I did this changes briefly just to be sure that the warning caused by the import statement.
I will fix this and all other same cases in a separate .ts PR